mockgen/parse.go

```diff
@@ -168,6 +185,7 @@ type fileParser struct {
 	auxInterfaces      *interfaceCache
 	srcDir             string
 	excludeNamesSet    map[string]struct{}
+	includeNamesSet    map[string]struct{}
```

Comment:
worth adding a comment here:

```suggestion
	includeNamesSet    map[string]struct{} // empty to include all
```